lua/neogit/status.lua

```diff
@@ -177,13 +177,24 @@ local function draw_buffer()
   end
 
   -- stylua: ignore
+  if git.repo.head.branch == "(detached)" then
+
+  output:append(
+    string.format(
+      "Head: (detached) at %s %s",
+      git.repo.head.abbrev,
+      git.repo.head.commit_message or "(no commits)"
+    )
+  )
+  else
   output:append(
     string.format(
       "Head:     %s %s",
       git.repo.head.branch,
       git.repo.head.commit_message or "(no commits)"
     )
   )
+  end
```

Comment:
I don't mind ignoring stylua here, but can you indent the inner blocks?

Reply:
Fixed, didn't see that there was a stylua ignore there above